patches/integration.diff

```diff
@@ -21,8 +21,12 @@ Index: code-server/lib/vscode/src/vs/server/node/server.main.ts
  import product from 'vs/platform/product/common/product';
  import * as perf from 'vs/base/common/performance';
  
-@@ -33,37 +33,42 @@ const errorReporter: ErrorReporter = {
- 	}
+@@ -30,41 +30,46 @@ const errorReporter: ErrorReporter = {
+ 
+ 	onDeprecatedOption: (deprecatedOption: string, message) => {
+ 		console.warn(`Option '${deprecatedOption}' is deprecated: ${message}`);
+-	}
++	},
```

Comment:
I assumed this wasn't something we deleted but something new added in 1.66, right?

Reply:
Looks like it, yeah.

Reply:
Er...or maybe not, looks like they do not have the trailing comma but we do?

Reply:
Ahhh, must have been my editor. I'll update this.